.pre-commit-config.py27.yaml

```diff
@@ -56,9 +52,11 @@ repos:
     rev: v1.4.4
     hooks:
     -   id: autopep8
+        args:
+        - --in-place
+        - --ignore=E124,E226,E24,W50,W690 # E124 conflicts with pylint
```

Comment:
I think those kinds of problems are going to be fixed in #34.

This specific one is a conflict between pylint and pep8. See https://github.com/PyCQA/pylint/issues/747 It seems that this correction is not the right one. But #34 has the same problem so we can fix it there.

Reply:
Gotcha -- not sure why this hasn't come up for us before, since these hooks are enabled on all of our other repos... what are all of these disables?

E124 in particular I'm ok with disabling -- I think we've generally defaulted to the pylint style anyway.

Reply:
Instead of doing that, we're going to disable `bad-continuation` in Pylint, because Pylint is wrong in that case. See the last part of [the indentation section of PEP-8](https://www.python.org/dev/peps/pep-0008/#indentation):

> The closing brace/bracket/parenthesis on multiline constructs may either line up under the first non-whitespace character of the last line of list [...]

(the other case is not relevant)

So I think we can remove these 3 lines and ignore `bad-continuation` in Pylint instead.

cc @gunjanswitchco

Reply:
I could go either way on accepting the pylint of the autopep8 versions here; I would say pick whichever one our code _already_ conforms to, since in this case I think its more important to have the standard than exactly which one it is. ++ for picking one and rolling with it :+1: